ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -191,6 +191,29 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-byo-route53
+- as: aws-ipi-compact-p2-f14
+  cron: 4 21 18,2 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @fips'
+      TAG_VERSION: '@4.13'
+      TEST_ADDITIONAL: Network_Observability
```

Comment:
Is this intentionally ? Any context why we need to test `Network_Observability` on profile `aws-ipi-compact` ?

Reply:
I just coped the lines from other existing jobs, I am going to remove it.

Reply:
@jianlinliu AWS part is being added by #36163

Reply:
oh, good to know, I am going to drop aws part in this PR.